lib/dynamo/dynamo/jobs.py

```diff
@@ -118,6 +143,34 @@ def _prepare_job_for_database(
     return prepared_job
 
 
+def _prepare_archive_job_for_database(job: dict, user_id: str, request_time: str, product: asf.ASFProduct) -> dict:
+    prepared_job = {
+        'job_id': str(uuid4()),
```

Comment:
Let's factor out a `get_job_id` function and use it in both of the prepare job functions.